Packs/Claroty/Integrations/Claroty/Claroty.yml

```diff
@@ -29,32 +29,42 @@ configuration:
   required: false
   type: 8
 - defaultvalue: 7 days
-  display: The initial time to fetch from
+  display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
   hidden: false
-  name: fetch_time
+  name: first_fetch
```

Comment:
Changing the name of a function param is breaking bc.
In this case, what we usually do is hide the existing integration param and create a new one with a different name.
However, in your case, I don't see any reason to change the name field (such as changing the type or such).
therefore, I think you should consider going back to the old name (the display name is fine that way.)